awx/main/tests/functional/models/test_schedule.py

```diff
@@ -0,0 +1,143 @@
+from awx.main.models import JobTemplate, Schedule
```

Comment:
Reviewers: please help me think of other wacky test cases to add here.

Reply:
See some of my `old` tests here: https://github.com/ansible/awx/blob/1.0.0/awx/main/tests/old/schedules.py

Reply:
Gonna replicate some of your "infinite recurrence" tests.